docs/source/hooks/common_use_cases.md

```diff
@@ -16,7 +16,7 @@ def say_hello(node: Node):
 Then you can either add it to a single node based on the node's name:
 
 ```python
-# <your_project>/src/<your_project>/hooks.py
+# <project-name>/src/<project-name>/hooks.py
```

Comment:
```suggestion
# <project_root>/src/<package_name>/hooks.py
```
(occurs multiple times in this file and other files)

Or even better I think just this will suffice since it's obvious here you're already in the project:
```suggestion
# src/<package_name>/hooks.py
```

Just to clarify, let's say I have `project_name = "My project"`. This would by default generate `package_name = "my_package"`. This is the name of the folder inside `src` and must be a valid Python package name. See https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/cookiecutter.json for the logic that does this. What we call `package_name` here in documentation is the same as what we call `python_package` in cookiecutter.json.

Reply:
Thanks a lot for the clarification, these changes are in [344a6fb](https://github.com/kedro-org/kedro/pull/1753/commits/344a6fb49d232371b8f592ee5ccc35d681bee043).

Reply:
Ah yes, I see. It won't make much difference in reality but I actually think these would make slightly more sense anyway if they used `package_name = metadata.package_name` rather than `project_name` as they currently do. Please could you change those and then the relevant `<project_name>` to `<package_name>`? Then we won't have any `<project_name>` left at all which would be nice.

Reply:
Made that change, no more instances of <project_name> left in the docs :).